kong/llm/drivers/shared.lua

```diff
@@ -18,8 +19,10 @@ local log_entry_keys = {
   PROVIDER_NAME = "ai.meta.provider_name",
 }
 
+local openai_override = os.getenv("OPENAI_TEST_PORT")
```

Comment:
This logic can be slightly simplified with this line and the one below
```suggestion
local openai_port = os.getenv("OPENAI_TEST_PORT") or 443
```